### ISSUE TITLE
Rename AOI to PU?

### ISSUE THREAD

**Issue by DanWismer** [NCC-CNC/wtw-data-prep#4]
I am wondering if we should start using the planning unit jargon to describe our grid? 

For example, the AOI.shp is the polygon needed to create the PU.shp, and PU.tif. This way we have a clear separation of the input needed (an AOI shapefile) and the outputs generated (vector and raster PU grid)

**Comment by edwardsmarc** [comment]
This is a great idea. I started to realise this as I was writing the readMe but didn't quite finish the thought.

How about the following changes:

- What I've called `AOI/AOI_polygon.shp` would become `AOI/AOI.shp`. It's just a renamed version of the user provided AOI polygon. Any reference to AOI relates to the AOI polygon and not the grid.

- `AOI.shp`, `AOI.tif` and `AOI0.tif` would become `PU.shp`, `PU.tif` and `PU0.tif`. Planning units only exist within the WTW environment. Any reference to planning units relates to the actual grid to be used in WTW.

- Any reference to the NCC grid would remain the same, but we don't refer to the NCC grid as planning units because it's never used within WTW. It's just a dataset used for prep.

I can add these definitions to the top of the readMe.
